components/identity-mgt/org.wso2.carbon.identity.mgt/src/main/java/org/wso2/carbon/identity/mgt/store/UserStoreBasedIdentityDataStore.java

```diff
@@ -123,9 +123,13 @@ public UserIdentityClaimsDO load(String userName, UserStoreManager userStoreMana
                         userDataMap.put(claimUri, claim.getValue());
                     }
                 }
+            } else {
+                // null is returned when the user doesn't exist
+                return null;
             }
         } catch (UserStoreException e) {
-            // ignore may be user is not exist
+            log.error("Error while reading user claim values");
```

Comment:
exception is not printed. correct way is as following.

log.error("Error while reading user claim values", e);